### PR TITLE
751 Changing access to the Student tab and functionality of certain roles 

### PR DESCRIPTION
**Pull request by AccorDDD**

### CODE REVIEW

File: CharlieBackend.Panel/Views/Students/AllStudents.cshtml [hunk 1]
```diff
@@ -68,6 +68,7 @@
 
                                     if (item.IsActive)
                                     {
+
```
Comment (BobTheSuperman) [inline CharlieBackend.Panel/Views/Students/AllStudents.cshtml:71]: Unnecessary line break

**Review by BobTheSuperman — APPROVED** [review]
Looks good

File: CharlieBackend.Panel/Views/Students/AllStudents.cshtml [hunk 2]
```diff
@@ -1,12 +1,12 @@
-﻿
-@model IEnumerable<CharlieBackend.Panel.Models.Students.StudentViewModel>
-
+﻿@model IEnumerable<CharlieBackend.Panel.Models.Students.StudentViewModel>
 @{
     Layout = "_Layout";
 }
```
Comment (DmitryTkachenko) [inline CharlieBackend.Panel/Views/Students/AllStudents.cshtml:7]: Unnecessary space

File: CharlieBackend.Panel/Views/Students/AllStudents.cshtml [hunk 3]
```diff
@@ -1,12 +1,12 @@
...
```
Comment (marinapelin) [inline CharlieBackend.Panel/Views/Students/AllStudents.cshtml:5]: please remove 2 lines and everywhere else

**Review by ArsenMaistryshyn — APPROVED** [review]

**Review by Ssashaa — APPROVED** [review]

**Review by DmitryTkachenko — APPROVED** [review]

**Pull request merged**